docs/chapters/workflow/release/release-process.rst

```diff
@@ -15,7 +15,7 @@ Binary images (starting from Pelux 3.0) are now available for `download`_. Each
 
 Currently 4 different variants are released. Every hardware offers the qtauto image as well as the minimal image, while QEMU offers only the minimal image. 
 
-Besides the images, SDK is also released with the following format: 
+Besides the images, SDK is also released with the following format:
```

Comment:
Still confusing. SDK installer which name has the following format or something like that?

Reply:
Ok I can fix more stuff in the same file, but my focus was only to fix the paragraph referred to in the issue.

Reply:
I just find the phrase `SDK is also released with the following format` confusing: it sounds like it's talking about the SDK format, when in fact, it describes installer's name format.

Reply:
I tried to rephrase it. I hope it's good now.